docs/evaluation/how_to_guides/evaluation/use_prebuilt_llm_evaluator.mdx

```diff
@@ -0,0 +1,128 @@
+
+### Score configuration
+
+You can configure the score of your feedback to either be continuous or categorical. Here is how to do each of those:
+
+#### Continuous
+
+A continuous score can be helpful for evaluating something along a spectrum such as verbosity, creativity, or fluency to name a few.
+To define a continuous evaluator score, use the following code:
+
+```python
+from langsmith.llm_evaluator import ContinuousScoreConfig
+
+score_config=ContinuousScoreConfig(
+    key="conciseness", # the feedback key
+    description="The concicesness of the response from 0 to 10, with 0 being the most verbose possible and 1 being an answer with no superfluous words.", # the description of the "score" key in the structured output provided to the LLM
+    min=0, # defaults to 0
+    max=10, # defaults to 1
+)
+```
+
+#### Categorical
+
+A categorical score can be used to evaluate things such as whether the output contains any hallucinations, or if the output contains the same semantic meaning as the expected answer. 
+To define a categorical evaluator score, use the following code:
+
+```python
+from langsmith.llm_evaluator import CategoricalScoreConfig
+
+score_config=CategoricalScoreConfig(
+    key="correctness", # the feedback key
+    description="Whether or not the answer is correct based on the expected output, Y if correct, N if incorrect.", # the description of the "score" key in the structured output provided to the LLM
+    choices=["Y","N"] # the allowed values for the LLM judge to return
+)
+```
+
+### Explanation calibration
+
+If you want to include an explanation along with your score, you need to pass in `include_explanation=True` to your score config.
+You can additionally pass in the `explanation_desription` param to instruct the LLM on how to construct it's explanation. Below is a code example:
+
+```python
+from langsmith.llm_evaluator import CategoricalScoreConfig
+
+score_config=CategoricalScoreConfig(
+    key="bias", # the feedback key
+    description="Whether or not the answer is biased.", # the description of the "score" key in the structured output provided to the LLM
+    choices=["BIASED","NOT BIASED"] # the allowed values for the LLM judge to return,
+    include_explanation=True, # defaults to False
+    explanation_description="What bias, if any, was displayed in the answer"
+)
+```
+
+## Using LLMEvaluator with evaluate/aevaluate
+
+Putting it all together, you can use the following code to evaluate question/answer pairs to see if the LLM application produced an answer that matches the expected answer:
+
+
+```python
+from langchain_openai import ChatOpenAI
+from langsmith.evaluation import evaluate
+model = ChatOpenAI(name="gpt-4o-mini")
+client = Client()
+
+evaluator = LLMEvaluator(
+    prompt_template="Is the response correct? Compare it only to the expected response, don't compare it to any of your existing knowledge.\n<response>\n{content}\n</response>\n\n<expected>\n{expected}\n</expected>",
```

Comment:
Do we have off the shelf ones here? Or do you have to build the prompt yourself completely?

Reply:
currently you have to build yourself. @agola11 and I talked about perhaps creating some prebuilt instances of LLMEvaluator that took in a model choice and a variable mapping (since people probably have different keys/structures in their datasets). Do you think that's a good idea?

Reply:
Yeah I think that's what people probably want